ext/config/public/services.d/bootstrap.cfg

```diff
@@ -0,0 +1,5 @@
+# To enable the CA service, leave the following line uncommented
+puppetlabs.services.ca.certificate-authority-service/certificate-authority-service
+# To disable the CA service, comment out the above line and uncomment the line below
+#puppetlabs.services.ca.certificate-authority-disabled-service/certificate-authority-disabled-service
+
```

Comment:
I think we should call this file `ca.cfg`.  Also I think I'd probably vote for using `system` and, maybe, `user` instead of `public` and `private`, as per @nathanielksmith 's suggestion.